### PR TITLE
Remove debug from builds and tests

### PR DESCRIPTION
**Pull request by arbourd**
The info and above errors should be more than enough for debugging
failures in GitHuB Actions CI.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -136,8 +140,7 @@ jobs:
             --openssl \
             --njs \
             --cc-opt="-I njs/src/ -I njs/build"  \
-            --ld-opt="-L njs/build" \
-            --debug
+            --ld-opt="-L njs/build"
 
       - name: Make unit
         run: |
```
Comment (ac000) [inline .github/workflows/ci.yml:146]: Looks good. But perhaps this should be split out into own commit separate from the language updates...

Comment (arbourd) [reply .github/workflows/ci.yml:146]: I'm messing around with it to force a failure, but ya I agree! I'll undraft this when it's good to go.

**Comment by ac000** [comment]
LGTM

**Review by ac000 — APPROVED** [review]
